api/docs/http_catalog.md

```diff
@@ -33,13 +33,15 @@ The examples below use the POST method.
 
 Six parameters should be provided to the POST or GET:
 
-- `environment`: the environment name
-- `facts_format`: must be `pson`
-- `facts`: serialized pson of the facts hash.  One odd note: due to a long-ago misunderstanding in the code, this is
-doubly-escaped (it should just be singly-escaped).  To keep backward compatibility, the extraneous
+- `environment`: the environment name.
+- `facts_format`: must be `pson`.
+- `facts`: serialized pson of the facts hash. One odd note: due to a long-ago misunderstanding in the code, this is
+doubly-escaped (it should just be singly-escaped). To keep backward compatibility, the extraneous
```

Comment:
As the sad soul who introduced those two spaces after a full-stop, I salute you and :+1: this noble effort.